core/editor/editor.php

```diff
@@ -496,6 +496,7 @@ public function enqueue_scripts() {
 				'jquery-hover-intent',
 				'nouislider',
 				'pickr',
+				'react',
```

Comment:
?

Reply:
Until now it was included via webpack, but for caching for both editor & app frames - it's better to include it as an external file.